file-formats/fugr/type/zif_fb_aff_fugr_v1.intf.abap

```diff
@@ -0,0 +1,53 @@
+INTERFACE zif_fb_aff_fugr_v1
+  PUBLIC.
+
+  "! <p class="shorttext">Status</p>
+  "! status
+  "! $values {@link zif_fb_aff_fugr_v1.data:co_status}
+  TYPES ty_status TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Status</p>
+    "! status
+    BEGIN OF co_status,
+      "! <p class="shorttext">SAP Program</p>
+      "! SAP program
+      sap_program      TYPE ty_status VALUE 'P',
+      "! <p class="shorttext">Customer Program</p>
+      "! Customer Production Program
+      customer_program TYPE ty_status VALUE 'K',
+      "! <p class="shorttext">System Program</p>
+      "! System program
+      system_program   TYPE ty_status VALUE 'S',
+      "! <p class="shorttext">Test Program</p>
+      "! Test program
+      test_program     TYPE ty_status VALUE 'T',
+      "! <p class="shorttext">Not Classified</p>
+      "! Not classified
+      not_classified   TYPE ty_status VALUE space,
+    END OF co_status.
```

Comment:
Just as comment: This might become something reusable in the future, because it is also needed for programs

Reply:
Then, we can move it to the interface `zif_aff_types_v1` with the reusable types.

Reply:
yes

Reply:
> Then, we can move it to the interface `zif_aff_types_v1` with the reusable types.

I had in mind whether we have a reusable type for ABAP programs. I am not sure whether we should add it to `zif_aff_types_v1`